NitroxPatcher/Patches/Dynamic/ResourceTracker_OnPickedUp_Patch.cs

```diff
@@ -0,0 +1,27 @@
+using System.Reflection;
+using NitroxClient.GameLogic;
+using NitroxModel.Core;
+using NitroxModel.Helper;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Event when someone picks up an item.
+/// </summary>
+public sealed partial class ResourceTracker_OnPickedUp_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((ResourceTracker d) => d.OnPickedUp(default));
+    
+    public static bool Prefix(ResourceTracker __instance, Pickupable p)
+    {
+        TechType instanceTechType = __instance.techType;
+        GameObject instanceGameObject = __instance.gameObject;
+        if (instanceTechType != null && instanceGameObject != null)
+        {
+            var ITEM_SERVICE = NitroxServiceLocator.LocateService<Items>();
+            ITEM_SERVICE.PickedUp(instanceGameObject, instanceTechType);
```

Comment:
I think you should be able to call this directly like other patches do without having to assign it first.

```suggestion
            Resolve<Items>().PickedUp(instanceGameObject, instanceTechType);
```